zoltar-api/src/main/java/com/spotify/zoltar/Predictors.java

```diff
@@ -93,7 +93,7 @@ private Predictors() {
   @SuppressWarnings("checkstyle:LineLength")
   public static <ModelT extends Model<?>, InputT, VectorT, ValueT> PredictorBuilder<ModelT, InputT, VectorT, ValueT> newBuilder(
       final ModelLoader<ModelT> modelLoader,
-      final FeatureExtractor<InputT, VectorT> featureExtractor,
+      final FeatureExtractor<ModelT, InputT, VectorT> featureExtractor,
```

Comment:
Do we need to thread the entire model object through, or can we just pass the ID so we don't require the extra type param?

Reply:
we could, the reason is just to be consistent with the predict functions.

At some point, I was thinking of having a execution `Context` object that would encapsulate this ... but i decided to just go with model for now.